packages/drivers/odsp-driver/src/test/tsconfig.json

```diff
@@ -12,4 +13,3 @@
 			},
 		],
 	}
-	
```

Comment:
accidental end line removal

Reply:
its not letting me add it :(

Reply:
What isn't letting you?
Actually, I think this is probably a trick of the diff UI. When I look at the file locally I see that it is improperly indented. The last line has a tab. So, I expect that locally you have tab removed but still a blank line and that causes the funny UI. I checked with another diff tool and see it is tab removal:
![image](https://github.com/microsoft/FluidFramework/assets/30939149/e16585ad-4c85-4454-a586-48b6bd8acd97)
But also I see there is extra indentation. Can you remove the extra indent for the file (either now or in next PR)?